sdk/canton/BUILD.bazel

```diff
@@ -44,7 +44,7 @@ case object BuildInfo {{
   val scalaVersion: String = "{scala_version}"
   val sbtVersion: String = "bazel"
   val damlLibrariesVersion: String = "{sdk_version}"
-  val protocolVersions = Seq()
+  val protocolVersions = Seq("30")
```

Comment:
@paulbrauner-da @remyhaemmerle-da I don't think we should continue to have this file encoded in a string instead of reading it from the repo: it creates many issues.